localisation/text_l_english.yml

```diff
@@ -7920,6 +7920,14 @@
  FERVOR:0 "Fervor"
```

Comment:
Komentarz ogólny:

Ogólnie dobra robota, wydaje się, że wszystko działa.
Rzeczy, których brakuje:

- Lokalizacja dla opisu partii (automatycznie zaciągnęło przygotowaną wcześniej przeze mnie wersje z TODO, ale trzeba to uzupełnić.
- Centrum Reformacji (Lokalizacja jest z tego co pamiętam gotowa, ale centrum reformacji trzeba zdefiniować osobno dla każdej religii - wystarczy kopia już istniejących + zmiana nazwy)